k9mail/src/main/java/com/fsck/k9/K9.java

```diff
@@ -210,11 +210,17 @@
         WHEN_IN_LANDSCAPE
     }
 
+    public enum ContactViewMode {
+        EMAIL,
+        NAME,
+        BOTH
```

Comment:
Without the context of the other enum values it's not immediately obvious what `BOTH` means. Please rename to `NAME_AND_EMAIL`.